static/backbone/css/map.css

```diff
@@ -60,13 +72,28 @@ body { overflow: hidden; }
 	color: #444;
 	display: block;
 }
-
-.item:hover .close {
+.data-item {
+	padding: 2px 6px 2px 10px;
+	cursor: pointer;
+	border-bottom: solid 1px #eee;
+}
+.data-item:hover .close {
 	display: block !important;
 }
-.item:hover {
+.data-item:hover {
 	background-image: url(../images/bg.png);
 }
+.layer-item {
+	padding: 2px 6px 0px 20px;
+	color: #666;
+}
+.layer-item .item-icon {
+	margin: 0px !important;
```

Comment:
An aside, though this is fine for now.  The !importants indicate our stylesheet/css is getting kind of unmanageable.  At some point we should move to less or something, since !important can come back to bite you later